### ISSUE TITLE
Known Bug: deleting the default jenkins config

### ISSUE THREAD

**Issue by terabyte** [palantir/stashbot#10]
It has been reported that if you create a new jenkins service config, then delete the default one, the config page will start throwing exceptions with a stack trace including soy errors.  This issue is under investigation now.

**Comment by clalbus** [comment]
Ah, wish I knew about this earlier. I ran into it right now with Stashbot 1.3.8, is there a fix for this?

**Comment by srinivaschary** [comment]
I would like to know the fix for this or a workaround to get into the admin page? With uninstall and install the add-on again is also no luck.

**Comment by mschary932** [comment]
if anyone blocked on this, below is the workaround to get a fix:
connect to stash database and insert one entry in AO_B11572_JSCONFIG001 table:

INSERT INTO AO_B11572_JSCONFIG001 (ID, NAME) VALUES (1, "default");

**Comment by terabyte** [comment]
Yeah, sorry I didn't reply to this sooner, btw, mschary932's workaround should get you out of this situation.
